plugins-compat-tester-cli/pom.xml

```diff
@@ -13,6 +13,7 @@
   <description>Jenkins Plugin Compatibility Tester (PCT) against latest released version</description>
   
   <build>
+    <finalName>${project.artifactId}</finalName>
```

Comment:
> This field is generally almost useless given it will not be used for deployments and many other cases, but I guess that's fine for local dev.

This is the only usage of this field in Maven imo: the version is really important when you get out of the current dev cycle. But when you stay in the dev cycle, you can get ride of it. 

I use this when I'm using the jar in a Docker image. 

When you depoy the jar, you *need* the version to be present, to make sure users of the CLI can find the version quickly.

> Note that BTW we can probably achieve the same using an additional step in the Makefile, but again fine by me.

We could, but this is a built-in way to do it ;)